plugins/wacom-usb/fu-wac-plugin.c

```diff
@@ -39,10 +39,16 @@ fu_wac_plugin_init(FuWacPlugin *self)
 }
 
 static void
-fu_wac_plugin_constructed(GObject *obj)
+fu_wac_plugin_constructed_early(GObject *obj)
```

Comment:
rather than `plugin_constructed_early` could this be `object_constructed` perhaps?

Reply:
I think they should all have the fu_xxx_plugin prefix -- but fu_xxx_plugin_object_constructed would be okay too?

Reply:
Fixed.